build/images/flow-aggregator/Dockerfile

```diff
@@ -7,14 +7,16 @@ COPY . /antrea
 
 # Make sure the flow-aggregator binary is statically linked.
 RUN CGO_ENABLED=0 make flow-aggregator
+RUN make antctl-ubuntu
 
-FROM scratch
+FROM ubuntu:20.04
```

Comment:
maybe add a comment saying that we chose this base image so that a shell is available for users to exec into the container and run antctl easily.

Reply:
I would also add troubleshooting as another reason to run tools like pprof etc.

Reply:
Done